### DISCUSSION TITLE
Suddenly Links aren't working on new Deploys - even with old builds that used to work

### DISCUSSION THREAD

**Discussion by adam-b-marsh** [vercel/community#7272] [Help] [answered]
### Summary

I pushed a new build to github today and it deployed to vercel correctly.
However when ever i try to navigate using one of my Links - it doesnt work.
The URL changes - but it just stays on the page that was originally loaded.
In developer tools - nothing happens on the networking tab.

Everything works fine locally - on every commit i've tried pulling down from github.

Ive included a link to a GIF showing the issue.

[https://imgur.com/a/z7159oS](https://imgur.com/a/z7159oS)

Any help would be super appreciated!!

### Example

https://imgur.com/a/z7159oS

### Steps to Reproduce

Console of failing production deployment: 
 (the 404 for the ecommerce is unrelated)
![image](https://github.com/vercel/community/assets/86686502/0ec96c48-ad94-4a02-b17c-5a96b21b0b8c)

**Comment by amyegan** [comment]
Hey @adam-b-marsh! I need to know more about your project. 

Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that I can use to help you debug?

**Reply by adam-b-marsh** [reply, marked as answer]
@amyegan thank you for responding. I've determined this is not a Vercel issue. It seems to be a difference between my locally hosted supabase and the development version i have running hosted with supabase.

If i point the deployed vercel branches at my local instance via ngrok tunnel - they work fine.

I just have to figure out what the difference is between the two supabase instances it seems.

Thanks again.

[a comment by an automated account is not shown]
